numpy/typing/tests/test_typing.py

```diff
@@ -36,6 +35,7 @@
 if TYPE_CHECKING:
     # We need this as annotation, but it's located in a private namespace.
     # As a compromise, do *not* import it during runtime
+    from collections.abc import Iterator
```

Comment:
I believe that the comment above applies to `ParameterSet`, so it might be better to place this import above it

Reply:
Indeed, sorry about that. Fixed.